operations/app/src/environments/tf

```diff
@@ -147,8 +165,8 @@ if [[ "${ACTUAL_RESULTS?}" == "${EXPECTED_RESULTS?}" ]]; then
         echo -e "${WHITE?}Executing terraform ${TFCMD?}${NORMAL?}..."
     fi
     case "${TFCMD?}" in
-    graph|output|state*)
-        # Graph & output are special in that they do not like arguments
+    graph|output|validate|state*)
```

Comment:
why the trailing `*`? I could now to `./tf --cmd statefoo --cmd test` and it would drop into this branch...
same for `import`, `plan-file` (but _not_ for `*)`...)

Reply:
Fixed in next push